.github/workflows/release.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Create Release Pull Request or Publish to npm
         uses: changesets/action@v1.3.0
         with:
-          publish: yarn changeset publish
+          publish: yarn workspace @local/package-chores exe scripts/remove-eslint-from-templates.ts && yarn changeset publish
```

Comment:
Can't we call this script via [prepublishOnly](https://docs.npmjs.com/cli/v8/using-npm/scripts#prepare-and-prepublish) in package.json instead? Then it will happen consistsently whether publishing from CI or not.

Reply:
Seems that `yarn workspace @local/package-chores exe scripts/remove-eslint-from-templates.ts` works from `packages/block-template-xyz`. That's good. We can probably tweak the script so that it removes ESLint from `cwd` rather than for a predefined list of packages. 

[Asana task](https://app.asana.com/0/1203007126736604/1203132574912424/f) (internal)